setup.py

```diff
@@ -4,18 +4,18 @@
 import setuptools
 
 setuptools.setup(
-    name='jinahub-MY-DUMMY-EXECUTOR',
+    name='jinahub-text-sentencizer',
     version='1',
     author='Jina Dev Team',
     author_email='dev-team@jina.ai',
-    description='This is my dummy executor',
-    url='https://github.com/jina-ai/EXECUTOR_REPO_NAME',
+    description='The text sentencizer splits texts into sentences',
+    url='https://github.com/jina-ai/executor-text-sentencizer',
     classifiers=[
         'Programming Language :: Python :: 3',
         'Operating System :: OS Independent',
     ],
-    py_modules=['jinahub.SUB_PACKAGE_NAME.MODULE_NAME'],
-    package_dir={'jinahub.SUB_PACKAGE_NAME': '.'},
+    py_modules=['jinahub.text.sentencizer'],
+    package_dir={'jinahub.text': '.'},
```

Comment:
This should follw 
jinahub.[indexer|encoder|segmenter|ranker|].module_name
so I think we should use segmenter?